extension-basics.md

```diff
@@ -4,9 +4,13 @@ title: Extension Basics
 permalink: /extension-basics/
 published: false
 tags: []
-contributors: [caitmuenster]
-last_updated_by: caitmuenster
-date: 2019-07-09 09:00:00
+contributors:
+  [
```

Comment:
It would be nicer (easier to read) to use the non-abbreviated list form here rather than the inline form (square brackets):

```yaml
contributors:
  - caitmuenster
  - rbrishabh
```